v2-components/Header/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useEffect, useState } from 'react';
+import Connector from 'containers/Connector';
+
+import {
+  NetworkNameById,
+  NetworkIdByName,
+  NetworkName,
+  NetworkId,
+} from '@synthetixio/contracts-interface';
+import Navigation from '../Navigation';
+
+const Header: FC = () => {
+  const { isWalletConnected, walletAddress, connectWallet, switchNetwork, isMainnet, network } =
+    Connector.useContainer();
+
+  const [localNetwork, setLocalNetwork] = useState<NetworkName>(
+    isMainnet
+      ? NetworkNameById[NetworkIdByName.mainnet]
+      : NetworkNameById[NetworkIdByName['mainnet-ovm']]
+  );
+
+  useEffect(() => {
+    setLocalNetwork(
+      isMainnet
+        ? NetworkNameById[NetworkIdByName.mainnet]
+        : NetworkNameById[NetworkIdByName['mainnet-ovm']]
+    );
+  }, [isMainnet]);
```

Comment:
Instead of `isMainnet` I think we want to look at `network.id`. Otherwise we wont have support for `goerli` and `optimism-goerli`

Reply:
Yeah - I haven't actually supported those in the Dropdown at the moment. I might put them in and then just push it back to designers and see what they think (plus not supported)

Reply:
hmm yea we probably dont want those network to be in the dropdown. Just make sure we can connect to the test networks.

Reply:
Understood